tests/e2e/server/routes.auth.mjs

```diff
@@ -262,10 +268,14 @@ export default function (app) {
   });
 
   app.get(authPaths.accounts, wait, async (req, res) => {
-    res.redirect(
+    const referrer = new URL(req.get('Referer'));
+    const additionalQueryParams =
       // eslint-disable-next-line max-len
-      'https://sdkapp.example.com:8443/authn-social-login-idm/?state=rtu8pz65dbg6baw985d532myfbbnf5v&code=4%2F0AY0e-g5vHGhzfggdAuIofxnblW-iR1Y30G5lN5RvbrU8Zv5ZmtUVheTzSX7YMJF_usbzUA&scope=email+profile+openid+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.profile&authuser=0&hd=forgerock.com&prompt=none',
-    );
+      'state=rtu8pz65dbg6baw985d532myfbbnf5v&code=4%2F0AY0e-g5vHGhzfggdAuIofxnblW-iR1Y30G5lN5RvbrU8Zv5ZmtUVheTzSX7YMJF_usbzUA&scope=email+profile+openid+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.profile&authuser=0&hd=forgerock.com&prompt=none';
+    const redirectUrl = `${referrer.href}${
+      referrer.href.includes('?') ? '&' : '?'
+    }${additionalQueryParams}`;
+    res.redirect(redirectUrl);
```

Comment:
Adding some intelligence to the redirect logic here. It's determining redirection from the referrer, instead of it being hard-coded.